### ISSUE TITLE
Network port for connection from Babyware does not open

### ISSUE THREAD

**Issue by marciprete** [ParadoxAlarmInterface/pai#191]
## Alarm system
EVO192, access mode IP150

## Environment
- HassOS on RPI4

## Git branch
dev

## Describe the bug
The network configuration doesn't seem to open the port 10000 to connect from babyware. 
I uninstalled and reinstalled the add on several time, changed the port trying different ones but it's never open, and I'm 100% sure because i check the open ports with nmap. 

## To Reproduce
1. Start script 
2. Set the "Host" port in the Network section of the cofiguration page
3. Nothing is logged, but the port is not available.

## Logs
Nothing relevant

**Comment by yozik04** [comment]
Did you enabled IP Interface in config?

**Comment by marciprete** [comment]
I'm not sure i understood your question, anyway, this is my configuration file:

`LOGGING_FILE: null
LOGGING_LEVEL_CONSOLE: 20
LOGGING_LEVEL_FILE: 40
LOGGING_DUMP_PACKETS: false
LOGGING_DUMP_MESSAGES: false
LOGGING_DUMP_STATUS: false
LOGGING_DUMP_EVENTS: false
CONNECTION_TYPE: IP
SERIAL_PORT: /dev/ttyUSB0
SERIAL_BAUD: 9600
IP_CONNECTION_HOST: 172.16.1.10
IP_CONNECTION_PORT: 10000
IP_CONNECTION_PASSWORD: paradox
IP_CONNECTION_SITEID: null
IP_CONNECTION_EMAIL: null
IP_CONNECTION_PANEL_SERIAL: null
IP_CONNECTION_BARE: false
LIMITS: {}
SYNC_TIME: true
PASSWORD: '0000'
MQTT_ENABLE: true
MQTT_HOST: core-mosquitto
MQTT_PORT: 1883
MQTT_USERNAME: mqtt
MQTT_PASSWORD: 
MQTT_RETAIN: true
MQTT_BIND_ADDRESS: 127.0.0.1
MQTT_BIND_PORT: 0
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE: true`

**Comment by yozik04** [comment]
Some more available settings: https://github.com/ParadoxAlarmInterface/pai/blob/master/config/pai.conf.example

The key to your problem is: `IP_INTERFACE_ENABLE`

**Comment by marciprete** [comment]
Thanks for your help. After a long journey, I finally made it. 
When I added the IP_INTERFACE_ENABLE parameter and saved the configuration enabling the port 10000, the addon complained that it was already in use. I also uninstalled and reinstalled the add on (both prod and dev versions) with no success.
I then switched to port 10001 but after the first attempt, the add on couldn't log in anymore, neither directly nor via Babyware.
Finally, I add in the PAI configuration these other two parameters:

IP_INTERFACE_BIND_ADDRESS: 0.0.0.0
IP_INTERFACE_BIND_PORT: 10000

and now it works perfectly.

Thank you very much for your support.
